app-constants.js

```diff
@@ -5,9 +5,12 @@
 const UserRoles = {
   BookingManager: 'bookingmanager',
   Administrator: 'administrator',
-  ConnectManager: 'Connect Manager'
+  ConnectManager: 'Connect Manager',
+  TopcoderUser: 'Topcoder User'
 }
 
+const TopCoderUserPermissionRole = UserRoles.TopcoderUser
```

Comment:
I guess there is no need to export this role separately, can use it as `UserRoles.TopcoderUser` inside the code